docs/squadcast.md

```diff
@@ -0,0 +1,68 @@
+# Use with Squadcast
+
+Syntax:
+
+`export SQUADCAST_REFRESH_TOKEN=<YOUR_SQUADCAST_REFRESH_TOKEN>`
```

Comment:
have a line stating refresh token can be passed as a flag or else as one can set the env

Reply:
We have added this